app/models/group.rb

```diff
@@ -13,11 +13,11 @@ def resources
   end
 
   def add_admin(user)
-    groups_users.create(user: user, admin: true)
+    groups_users.new(user: user, admin: true).save
```

Comment:
Okay, I remember! It was because of this:

```ruby
if @group.add_user(user)
  'Member was successfully added.'
else
  'Member already in group.'
end
```

`create` always return the object, even if it couldn't be saved. Using `.save` either returns `true` or `false` which I can use to show the appropriate message.